src/lib/storage/table.hpp

```diff
@@ -235,9 +236,9 @@ class Table : private Noncopyable {
    * With C++20 we will get std::atomic<std::shared_ptr<T>>, which allows us to omit the std::atomic_load() and
    * std::atomic_store() function calls.
    *
-   * For the zero_allocator, see the implementation of Table::append_chunk.
+   * For the ZeroAllocator, see the implementation of Table::append_chunk.
```

Comment:
Please adapt `table.cpp` as well:  ` // To avoid someone reading an incomplete shared_ptr<Chunk>, we (1) use the zero_allocator for the concurrent_vector,`

Reply:
This location is already apadted.

Reply:
Blind!